Add task solution

NikitaOrliak opened the pull request:
[DEMO LINK](https://nikitaorliak.github.io/react_dynamic-list-of-posts/)

To be honest, it was much easier and more interesting than the previous task :D

volodymyr-soltys97 commented on src/App.tsx:
```diff
@@ -8,37 +8,98 @@ import { PostsList } from './components/PostsList';
+  const [createNewComment, setCreateNewComment] = useState<boolean>(false);
+
+  useEffect(() => {
+    getUsersData().then(setUsers);
```
Add catch method after then in order to handle the error

volodymyr-soltys97 reviewed (changes requested):
Almost done!
Let's fix one moment

IvanFesenko reviewed (changes requested):
Dropdown works incorrect - check video. It should select user in that case
https://github.com/mate-academy/react_dynamic-list-of-posts/assets/61904995/57be9184-56cd-487a-94e4-ac9094a885fe

If field value is not valid it should not be able to add comment
https://github.com/mate-academy/react_dynamic-list-of-posts/assets/61904995/d19695a0-8b2b-4668-9a07-01a722b7d0b9
https://github.com/mate-academy/react_dynamic-list-of-posts/assets/61904995/7306dc61-6e40-4bda-a397-4e93cd9cee5c

maxim2310 reviewed (changes requested):
Almost there) Just one more thing needs to be fixed
if while deleting comment have error you must to return comment

NikitaOrliak commented:
> Almost there) Just one more thing needs to be fixed if while deleting comment have error you must to return comment

In the [Working version](https://mate-academy.github.io/react_dynamic-list-of-posts/) I don't see the error or return comment, moreover, in the README there is "Delete the commnet immediately not waiting for the server response to improve the UX.". I already created a catch for deletion, but then I checked it in the working version and read readme and deleted it.

Added it again :)

plodnikate reviewed (approved):
Good job on it!
